File: app/src/main/java/com/appchamp/wordchunks/ui/game/fragments/GameFinishedFragment.java
package com.appchamp.wordchunks.ui.game.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.appchamp.wordchunks.R;


public class GameFinishedFragment extends Fragment {

    public GameFinishedFragment() {
        // Requires empty public constructor
    }


    public static GameFinishedFragment newInstance() {
        return new GameFinishedFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        View root = inflater.inflate(R.layout.frag_game_finished, container, false);

        return root;
    }
}

File: app/src/main/java/com/appchamp/wordchunks/ui/aftergame/GameFinishedFragment.kt
package com.appchamp.wordchunks.ui.aftergame

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.appchamp.wordchunks.R
import kotlinx.android.synthetic.main.frag_game_finished.*
import org.jetbrains.anko.browse


class GameFinishedFragment : Fragment() {

    companion object {
        fun newInstance() = GameFinishedFragment()
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater?.inflate(R.layout.frag_game_finished, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        rlRateUs.setOnClickListener {
            context.browse("market://details?id=${activity.packageName}")
        }
    }
}
